.devcontainer/README.md

```diff
@@ -125,22 +133,3 @@ root@ecee08edef4b:/build/c/oai# cat clang.findings | egrep "android|bugprone|cer
       5 misc-redundant-expression
      15 misc-unused-using-decls
 ```
-
-
-
-## Sharp Edges
-
-- Your Docker Container Volume contains your work!
-  - Be careful not to delete the container until outstanding work is pushed to your personal Github Fork
-  - Note cleanup for disk space below - and the danger here
-  - TODO: How to re-attach a Volume if you close VS Code?
-- Docker running out of disk on Mac VM
-  - Docker just leaks resources like mad
-  - Eventually you'll get a "out of space" error
-  - Or an error about apt-get failure
-  - TODO: find good docker command to wipe everything
-
-## TODOs
-
-- Get the Docker image up to DockerHub?
-  - Want easy way to build from Dockerfile locally though..
```

Comment:
### Sharp edges
- One can also a local copy of the repo and only mount it. This is imho much safer and less resource hungry. It is documented now. So the first point should be mitigated.
- Docker image cleanup is also covered in the official documentation now in the troubleshooting section, including a cleanup command.

### TODOs
- My assumption was that having GHCR, DockerHub us superfluous.
- Build locally. :heavy_check_mark: 
-

Reply:
Thanks for the additional cleanup. LGTM.